src/main/java/api/kirana/register/transaction/controllers/TransactionControllers.java

```diff
@@ -1,131 +1 @@
-package api.kirana.register.transaction.controllers;
-
-import api.kirana.register.response.Response;
-import api.kirana.register.transaction.models.TransactionDTO;
-import api.kirana.register.transaction.service.TransactionService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.security.access.prepost.PreAuthorize;
-import org.springframework.web.bind.annotation.*;
-
-@RestController
-@RequestMapping("/api/transaction")
-public class TransactionControllers {
-    private final TransactionService transactionService;
-
-    @Autowired
-    public TransactionControllers(TransactionService transactionService) {
-        this.transactionService = transactionService;
-    }
-
-
-    /**
-     * lists all transactions along with their specific details
-     * @return
-     */
-    @PreAuthorize("hasAuthority('ADMIN')")
-    @GetMapping("/getAll")
-    public ResponseEntity<Response> getAllTransactions() {
-        Response response = new Response();
-        response.setData(transactionService.getAllTransactions());
-        response.setMessage("List of all transactions");
-        return new ResponseEntity<>(response, HttpStatus.OK);
-    }
-
-    /**
```

Comment:
create api for getAllTransactionsForUser, also use pageable to control the number of returned transactions